modules/oxide/src/less/theme/components/image-tools/image-tools.less

```diff
@@ -10,16 +10,21 @@
 //
 
 .tox {
-  .tox-image-tools {
-    width: 100%;
+  .tox-spacer {
+    flex: 1;
   }
```

Comment:
Should this be here? As this is specific to imagetools IIUC so it looks like it should be in the `.tox-image-tools` block below instead so it remains scoped to that?

Reply:
I was thinking it might be useful elsewhere, but now that it's a magic class I guess it should be scoped.